.github/workflows/test-warehouse.yml

```diff
@@ -101,4 +101,4 @@ jobs:
         working-directory: ${{ env.TESTS_DIR }}
         run: |
           dbt deps
-          python run_e2e_tests.py -t "${{ inputs.warehouse-type }}" --clear-tests "${{ inputs.clear-tests }}"
+          python run_e2e_tests.py -g -t "${{ inputs.warehouse-type }}" --clear-tests "${{ inputs.clear-tests }}"
```

Comment:
Temporarily doing this because I need to regenerate the seeds for this branch (I added a column)